backend/schema/corpora_schema.md

```diff
@@ -0,0 +1,323 @@
+# Corpora Schema
```

Comment:
Are we deprecating the term "corpora" at this point?

Reply:
you still need a name that differentiates from the more agnostic viewpoing cellxgene has.   Corpora is a fine short-hand for that, easier than "the more prescriptive schema we want all data to eventually be converted to so it is integrable."

Reply:
Yeah I'm open to pretty much any word, but I don't think we can s/corpora/cellxgene/ the way we have everywhere else because this definitely isn't a "cellxgene schema".